libs/backend/database/src/models/event/game.model.ts

```diff
@@ -265,3 +265,46 @@ export class Game extends Model<InferAttributes<Game>, InferCreationAttributes<G
   hasPlayers!: BelongsToManyHasAssociationsMixin<Player, string>;
   countPlayer!: BelongsToManyCountAssociationsMixin;
 }
+
+@InputType()
+export class GameNewInputPlayers {
+  @Field(() => ID)
+  id!: string;
+
+  @Field(() => ID)
+  systemId!: string;
+
+  @Field(() => Int)
+  team!: number;
+
+  @Field(() => Int)
+  player!: number;
+}
+
+@InputType()
+export class GameNewInput extends PartialType(
+  OmitType(Game, ['id', 'visualCode', "rankingPoints", "players", 'competition','linkId', 'linkType', "tournament", "createdAt", "updatedAt"] as const),
```

Comment:
I usually take the updatemodel as base so:

`OmitType(GameUpdateInput, ['id'] as const),`